base_layer/core/src/chain_storage/blockchain_database.rs

```diff
@@ -1229,6 +1229,8 @@ pub fn calculate_mmr_roots<T: BlockchainBackend>(db: &T, block: &Block) -> Resul
 
     let metadata = db.fetch_chain_metadata()?;
     if header.prev_hash != *metadata.best_block() {
+        error!(target: LOG_TARGET, "Block hash: {}", header.hash().to_hex());
+        error!(target: LOG_TARGET, "Block : {}", block);
```

Comment:
I think you meant to take these out / make into a more descriptive debug message